python/lsst/ts/ess/common/device/mock_device.py

```diff
@@ -92,15 +100,26 @@ def __init__(
         #     The sensor produces an error (True) or not (False) when being
         #     read.
         self.in_error_state = False
+        #     The sensor produces only noise telemetry (True) or not (False)
+        #     when being read. This applies only to LD-250 sensors.
```

Comment:
I'm sorry to be dense, but I don't understand what it means to "produce *only* noise (or strike) telemetry". Could you clarify the notes a bit?

Do you mean something like: "The sensor always (True) or never (False) produces noise telemetry"?

Would it be helpful to control the fraction of replies that have a detected noise or strike? (0 for never, 1 for every sample). Or are boolean flags all you really need? I can imagine fractions being useful for running in simulation mode, but perhaps not unit tests.

Reply:
In this state the sensor will only produce noise messages (which contain no data) which is what I intended to describe. I agree that it is not very clear so I adopted your sentence.